test/unit/cli-domain-init-template--initPackage.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const injectr = require('injectr');
+
+
+const deps = {
+  'cross-spawn': {
+    sync: sinon.spy()
+  }
+};
+
+const initPackage = injectr('../../src/cli/domain/init-template/initPackage.js', deps, {});
```

Comment:
This is a minor one, but I think we usually follow conventions of `init-package` for filenames. Are you able to bring all the new files in this PR back to follow the same convention?